eng/dockerfile-templates/runtime/Dockerfile.ltsc2016

```diff
@@ -0,0 +1,58 @@
+# escape=`
+
+FROM mcr.microsoft.com/windows/servercore:ltsc2016-amd64
+
+ENV COMPLUS_NGenProtectedProcess_FeatureEnabled=0
+
+RUN `
+{{if PRODUCT_VERSION != "4.6.2"
+:    # Install .NET Fx {{PRODUCT_VERSION}}
+    powershell -Command `
+{{if PRODUCT_VERSION = "3.5"
+:        $ErrorActionPreference = 'Stop'; `
+}}        $ProgressPreference = 'SilentlyContinue'; `
+        [Net.ServicePointManager]::SecurityProtocol = [Net.SecurityProtocolType]::Tls12; `
+        Invoke-WebRequest `
+            -UseBasicParsing `
+{{if PRODUCT_VERSION = "3.5"
+:            -Uri "https://dotnetbinaries.blob.core.windows.net/dockerassets/microsoft-windows-netfx3-ltsc2016.zip" `
+            -OutFile microsoft-windows-netfx3.zip; `
+        Expand-Archive microsoft-windows-netfx3.zip; `
+    && del microsoft-windows-netfx3.zip `
+    && dism /Online /Quiet /Add-Package /PackagePath:C:\microsoft-windows-netfx3\microsoft-windows-netfx3-ondemand-package.cab `
+    && rmdir /S /Q microsoft-windows-netfx3 `
+^else:{{
+    if PRODUCT_VERSION = "4.7"
+:            -Uri "https://download.microsoft.com/download/D/D/3/DD35CC25-6E9C-484B-A746-C5BE0C923290/NDP47-KB3186497-x86-x64-AllOS-ENU.exe" `
+^   elif PRODUCT_VERSION = "4.7.1"
+:            -Uri "https://download.microsoft.com/download/9/E/6/9E63300C-0941-4B45-A0EC-0008F96DD480/NDP471-KB4033342-x86-x64-AllOS-ENU.exe" `
+^   elif PRODUCT_VERSION = "4.7.2"
+:            -Uri "https://download.microsoft.com/download/6/E/4/6E48E8AB-DC00-419E-9704-06DD46E5F81D/NDP472-KB4054530-x86-x64-AllOS-ENU.exe" `
+^   elif PRODUCT_VERSION = "4.8"
+:            -Uri "https://download.visualstudio.microsoft.com/download/pr/7afca223-55d2-470a-8edc-6a1739ae3252/abd170b4b0ec15ad0222a809b761a036/ndp48-x86-x64-allos-enu.exe" `
+}}            -OutFile dotnet-framework-installer.exe `
+    && start /w .\dotnet-framework-installer.exe /q `
+    && del .\dotnet-framework-installer.exe `
+}}    && powershell Remove-Item -Force -Recurse ${Env:TEMP}\* `
+    `
+    # Apply latest patch
+    && powershell -Command `
+        $ProgressPreference = 'SilentlyContinue'; `
+        [Net.ServicePointManager]::SecurityProtocol = [Net.SecurityProtocolType]::Tls12; `
+        Invoke-WebRequest `
+            -UseBasicParsing `
+            -Uri "{{if PRODUCT_VERSION = "4.8":{{VARIABLES["lcu|ltsc2016"]}}^else:{{VARIABLES["lcu|ltsc2016|3.5-4.7.2"]}}}}" `
+            -OutFile patch.msu; `
+    && mkdir patch `
+    && expand patch.msu patch -F:* `
+    && del patch.msu `
+    && dism /Online /Quiet /Add-Package /PackagePath:C:\patch\windows10.0-{{if PRODUCT_VERSION = "4.8":{{VARIABLES["kb|ltsc2016"]}}^else:{{VARIABLES["kb|ltsc2016|3.5-4.7.2"]}}}}-x64{{if PRODUCT_VERSION = "4.8":-ndp48}}.cab `
```

Comment:
Hopefully the `-ndp48` suffix pattern holds going forward.  For example, in the 2020 5B release, the cab filename did not have a `-ndp48` suffix.

Reply:
Yes.  Hopefully a number of the conditionals will be eliminated as we go forward and there becomes less variance between the versions.  But that feels like an eternity away given how long LTSC versions are supported.